platform/s3ip-sysfs/scripts/s3ip_sysfs_conf.json

```diff
@@ -0,0 +1,76 @@
+{
+  "s3ip_syfs_paths": [
+    {
+      "path": "/sys_switch/temp_sensor",
+      "type" : "path",
+      "value" : "/sys/s3ip/temp_sensor",
+      "description": "温度传感器信息"
```

Comment:
These descriptions are being shown in Mandarin. Can you change it to english pls?